cluster/ci/config/prow/plugins.yaml

```diff
@@ -496,6 +519,7 @@ triggers:
   - kiegroup/kie-cloud-operator
   - containers/libpod
   - operator-framework
+  - redhat-developer/odo
```

Comment:
Remove this one -- you're in the next stanza already. This stanza will make it required to be in the `OpenShift` org to trigger jobs

Reply:
Done